backend/CircleForms/Controllers/Authorization/OAuth/OAuthController.cs

```diff
@@ -50,9 +59,26 @@ public async Task<IActionResult> NewCode([FromQuery(Name = "code")] string code,
             //TODO: do something with it
             if (await _usersService.Get(user.Id) == null)
             {
-                await _usersService.Create(user);
+                user = await _usersService.Create(user);
             }
 
+            var tokenHandler = new JwtSecurityTokenHandler();
```

Comment:
I think this can be moved out to a separate `JWTService` so that we don't pollute auth controller with JWT creation